src/index.html

```diff
@@ -17,6 +30,38 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="logo"
+        class="image"
+      />
+      <nav class="flex-text">
+        <ul>
+          <li>
+            <a
+              href="Apple"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li><a href="Samsung">Samsung</a></li>
+          <li><a href="Smartphones">Smartphones</a></li>
+          <li>
+            <a
+              href="Laptops&Computers"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li><a href="Gadgets">Gadgets</a></li>
```

Comment:
The `href` attribute for the 'Gadgets' link should be a valid URL. Consider using a relative or absolute URL, such as `href='/gadgets'` or `href='https://example.com/gadgets'`.